app/support/statement_pdf.rb

```diff
@@ -40,77 +36,11 @@ def filename
                                       invoice_number: @statement.invoice_number)
   end
 
-  def render
```

Comment:
Looks like I do need this method. I think it gets called when creating the PDF for attaching to an email.